tests/test_flows.py

```diff
@@ -661,9 +680,22 @@ def test_combined_infra_metrics(self):
                                                env.controller.mac, mid)
         debug("Check AP metrics response has AP metrics")
         ap_metrics_2 = self.check_cmdu_has_tlv_single(response, 0x94)
-        # TODO currently STA traffic stats are missing
-        # debug("Check AP metrics response has STA traffic stats")
-        # sta_traffic_stats_2 = self.check_cmdu_has_tlv_single(response, 0xa2)
+        if ap_metrics_2:
+            if ap_metrics_2.ap_metrics_bssid != vap2.bssid:
+                self.fail("AP metrics response with wrong BSSID {} instead of {}".format(
+                    ap_metrics_2.ap_metrics_bssid, vap2.bssid))
+
+        debug("Check AP metrics response has STA Link Metrics")
+        sta_metrics_2 = self.check_cmdu_has_tlv_single(response, 0x96)
```

Comment:
Test should check that response does not contain any Associated STA Traffic Stats TLV

Reply:
https://github.com/prplfoundation/prplMesh/issues/1340